### ISSUE TITLE
Brazilian Portuguese translation

### ISSUE THREAD

**Issue by floppyD** [Fr4gorSoftware/SecScanQR#136]
I noticed that you don't have a PT-BR translation, would you accept one?
```
<resources>
    <!--Activity labels -->
    <string name="app_name">SecScanQR</string>
    <string name="activity_generate">Gerador de QR-code</string>
    <string name="activity_history">Histórico</string>
    <string name="activity_history_details">Histórico</string>
    <string name="title_activity_geo_generator">Gerador de Geolocalização</string>
    <string name="title_activity_text_generator">Gerador de Texto</string>
    <string name="title_activity_barcode_generate">Gerador de Código de Barras</string>
    <string name="title_activity_vcard_generator">Gerador de VCard</string>
    <string name="title_activity_wifi_generator">Gerador de Wifi</string>
    <!-- MainActivity-->
    <string name="title_format">Formato do código:</string>
    <string name="title_information">Informação do código:</string>
    <string name="default_text_main_activity">Toque no botão "Digitalizar" para digitalizar o QR-code/Código de Barras</string>
    <string name="send_to">Compartilhar com:</string>
    <string name="xzing_label">Ligue a lanterna usando as teclas de volume</string>
    <string name="notice_clipoard">Copiado para a área de transferência</string>
    <!--MainActivity Errors-->
    <string name="error_scan_first">Primeiro digitalize um QR-code</string>
    <string name="error_canceled_scan">Você cancelou a digitalização</string>
    <string name="error_add_to_database">Não foi possível adicionar o QR-code ao histórico</string>
	<string name="error_file_not_found">Arquivo não encontrado</string>
    <string name="error_code_not_found">Código de barras/Qr-code não encontrado</string>
    <!-- MainActivity Navigation-->
    <string name="title_scan">Digitalizar</string>
    <string name="title_generate">Gerar</string>
    <string name="title_history">Histórico</string>
    <!-- MainActivity Action Navigation-->
    <string name="title_copy">Copiar</string>
    <string name="title_share">Compartilhar</string>
    <string name="title_reset">Limpar</string>
    <string name="title_openInWeb">Abrir na Internet</string>
    <string name="title_createContact">Criar Contato</string>

    <!-- GenerateActivity-->
    <string name="btn_save">Salvar</string>
    <string name="toast_save">Salvo na pasta "SecScanQR" no armazenamento interno</string>
    <string name="toast_permission_needed">Permissão necessária</string>
    <string name="toast_erorr">Ocorreu um erro</string>
    <string-array name="generators">
        <item>CÓDIGO DE BARRAS</item>
        <item>GEOLOCALIZAÇÃO</item>
        <item>TEXTO</item>
        <item>VCARD</item>
        <item>WIFI</item>
    </string-array>

    <!-- BarcodeGeneratorActivity-->
    <string name="activity_barcode_generator">Gerador de Código de Barras</string>
    <string-array name="barcode_formats_array">
        <item>BARCODE</item>
        <item>CODE_128</item>
        <item>CODE_39</item>
        <item>EAN_13</item>
        <item>EAN_8</item>
        <item>ITF</item>
        <item>PDF_417</item>
        <item>UPC_A</item>

    </string-array>

    <!-- TextGeneratorActivity-->
    <string name="hint_text2qr">Você pode digitar algum texto aqui</string>
    <string name="tv_format">Formato:</string>
    <string-array name="text_formats_array">
        <item>QR_CODE</item>
        <item>AZTEC</item>

    </string-array>
    <!-- TextGeneratorActivity Erorrs-->
    <string name="error_text_first">Primeiro digite algum texto</string>
    <string name="error_generate">Entrada inválida para este formato</string>

    <!-- GeoGeneratorActivity-->
    <string name="hint_latitude">latitude, exemplo: 43.73</string>
    <string name="hint_longtitude">longitude, exemplo: 10.73</string>
    <string-array name="formats_geo_array">
        <item>QR_CODE</item>
        <item>AZTEC</item>
    </string-array>
    <string name="cbLatitude">norte</string>
    <string name="cbLongtitude">leste</string>

    <!-- GeoGeneratorActivity Errors-->
    <string name="error_geo_first">Primeiro introduza valores válidos para latitude e longitude</string>

    <!-- VCARDGeneratorActivity-->
    <string name="hint_name">nome</string>
    <string name="hint_fn">primeiro nome</string>
    <string name="hint_org">organização</string>
    <string name="hint_web">página na internet</string>
    <string name="hint_email">e-mail</string>
    <string name="hint_mobil">celular</string>
    <string name="hint_tele_work">telefone(profissional)</string>
    <string name="hint_tele_private">telefone(pessoal)</string>
    <string name="hint_street">rua</string>
    <string name="hint_plz">código postal</string>
    <string name="hint_city">cidade</string>
    <string name="hint_state">estado</string>
    <string name="hint_country">país</string>

    <!-- VCARDGeneratorActivity Errors-->
    <string name="error_fn_or_name_first">Primeiro digite um nome ou primeiro nome</string>

    <!-- WifiGeneratorActivity -->
    <string name="cbHidden">Oculto</string>
    <string name="hint_ssid">SSID</string>
    <string name="hint_password">Senha</string>
    <string name="tv_encryption">Criptografia: </string>
    <string-array name="text_formats_encryption">
        <item>NENHUMA</item>
        <item>WEP</item>
        <item>WPA/WPA2</item>
    </string-array>

    <!-- HistoryActivity -->
    <string name="reset_database">Limpar</string>
    <string name="share_database">Compartilhar</string>
    <!-- HistoryActivity Delete Dialog -->
    <string name="delete_history_dialog_message">Todas as entradas serão excluídas!</string>
    <string name="delete_history_dialog_confirmation">OK</string>
    <string name="delete_history_dialog_cancel">Cancelar</string>

    <!-- HistoryActivity Errors-->
    <string name="error_not_in_database">Não foi encontrado no banco de dados do histórico</string>
    <string name="error_sqlexception">Este formato não é suportado para a função histórico</string>

    <!-- HistoryDetailsActivity-->
    <string name="notice_deleted_from_database">Excluído do histórico</string>
    <!-- HistoryDetailsActivity Action Navigation-->
    <string name="title_delete">Excluir</string>

    <!-- Optionsmenu-->
    <string name="settings_menu_about">Sobre</string>
    <string name="settings_settings">Configurações</string>

    <!-- AboutDialog-->
    <string name="about_dialog">Sobre</string>
    <string name="copyright">Aviso de Direitos Autorais</string>
    <string name="copyright_holder">Copyright© 2017-2019 T-Dankworth</string>
    <string name="github_link">Encontre SecScanQR no Github: https://github.com/Fr4gorSoftware/SecScanQR</string>
    <string name="license">Este aplicativo foi criado sobre os termos da Licença Pública Geral GNU. Para mais informações, consulte o link: https://www.gnu.org/licenses/gpl-3.0.en.html</string>
    <string name="logo_designer">O logotipo deste aplicativo foi criado por s-yous: https://github.com/s-yous</string>
    <string name="translator">Tradução Portuguesa por floppyD</string>
    <string name="zxing_license">Este aplicativo utiliza a biblioteca ZXING que se encontra sobre os termos da licença Apache 2.0. Para mais informações, consulte o link: http://www.apache.org/licenses/LICENSE-2.0</string>
    <!-- Strings related to Settings -->
    <string name="title_activity_settings">Configurações</string>
    <!-- Scanner settings -->
    <string name="pref_header_scanner">Digitalizador</string>
    <string name="pref_header_history_settings">Histórico</string>
    <string name="pref_header_search_engine_settings">Mecanismo de Busca</string>

    <string name="pref_title_camera">Câmera</string>
    <string-array name="pref_camera_entries">
        <item>camera traseira</item>
        <item>camera frontal</item>
    </string-array>
    <string-array name="pref_camera_values">
        <item>0</item>
        <item>1</item>
    </string-array>

    <string name="pref_title_auto_scan">Iniciar digitalizador automaticamente</string>
    <string-array name="pref_auto_scan_entries">
        <item>Ligado</item>
        <item>Desligado</item>
    </string-array>
    <string-array name="pref_auto_scan_values">
        <item>true</item>
        <item>false</item>
    </string-array>

    <string name="pref_title_auto_clipboard">Área de transferência inteligente</string>
    <string-array name="pref_auto_clipboard_entries">
        <item>Ligado</item>
        <item>Desligado</item>
    </string-array>
    <string-array name="pref_auto_clipboard_values">
        <item>true</item>
        <item>false</item>
    </string-array>

    <string name="pref_title_day_night_mode">Modo diurno/noturno (Versão beta)</string>
    <string-array name="pref_day_night_mode_entries">
        <item>Diurno (Necessário reiniciar)</item>
        <item>Noturno (Necessário reiniciar)</item>
    </string-array>
    <string-array name="pref_day_night_mode_values">
        <item>0</item>
        <item>1</item>
    </string-array>

    <string name="pref_title_history">Histórico (ligado/desligado)</string>
    <string-array name="pref_history_entries">
        <item>Ligado</item>
        <item>Desligado</item>
    </string-array>
    <string-array name="pref_history_values">
        <item>true</item>
        <item>false</item>
    </string-array>

    <string name="pref_title_search_engine">Motor de Busca</string>
    <string-array name="pref_search_engine_entries">
        <item>Bing</item>
        <item>DuckDuckGo</item>
        <item>Google</item>
        <item>Qwant</item>
        <item>Qwant Lite</item>
        <item>Startpage</item>
        <item>Yahoo</item>
        <item>Yandex (RUS)</item>
    </string-array>
    <string-array name="pref_search_engine_values">
        <item>1</item>
        <item>2</item>
        <item>3</item>
        <item>4</item>
        <item>5</item>
        <item>6</item>
        <item>7</item>
        <item>8</item>
    </string-array>

    <string name="pref_title_barcode_search_enigne">Motor de Busca para Código de Barras</string>
    <string-array name="pref_barcode_search_engine_entries">
        <item>Nenhum motor de busca extra para Código de Barras</item>
        <item>Open Food Facts</item>
        <item>CodeCheck</item>
    </string-array>
    <string-array name="pref_barcode_search_engine_values">
        <item>0</item>
        <item>1</item>
        <item>2</item>
    </string-array>

</resources>

```

**Comment by Fr4gorSoftware** [comment]
Yeah of course. I will implement this for the next release. Keep in mind your devices has to be set on PT-BR to see the translation.

**Comment by Fr4gorSoftware** [comment]
Implemented in the current development version and will be available in version 1.2.6 for all users. Thank you for providing translations to the project :-)
